README.md

```diff
@@ -6,33 +6,66 @@
 [![Discourse status](https://img.shields.io/discourse/https/meta.discourse.org/status.svg?style=for-the-badge)](https://forums.habitat.sh)
 [![Open Source Helpers](https://www.codetriage.com/habitat-sh/habitat/badges/users.svg)](https://www.codetriage.com/habitat-sh/habitat)
 
-Want to try Habitat? [Get started here](https://www.habitat.sh/learn/).
+<img src="https://github.com/habitat-sh/habitat/blob/master/www/source/images/habitat-logo.png" width="150">
 
-Habitat is an application automation framework that allows you to build applications that have automation built-in. This provides modern applications that:
+[Habitat](http://habitat.sh) is open source software that creates platform-independent build artifacts and provides built-in deployment and management capabilities.
 
-* Provide repeatable builds
-* Run from single, immutable assets
-* Allow for runtime configuration for multiple deployment scenarios
-* Are agnostic to operating environment (works the same on bare metal, virtualization, containers, PaaS)
-* Provide idempotent behavior (the same inputs to the same asset provide the same outcome)
-* Provide convergent behavior (each service makes progress towards the correct behavior in the face of failure)
-* Expose promises to those who rely on it
-* Provide a low barrier to entry
-* Are language agnostic
+The goal of Habitat is to allow you to automate your application behavior when you create your application, and then bundle your application with the automation it needs to behave with the correct run time behavior, update strategies, failure handling strategies, and scaling behavior, wherever you choose to deploy it.
 
-To learn more about Habitat, please visit the [Habitat website](https://www.habitat.sh).
+## Overview and Demo
+[![Build, Deploy and Manage with Habitat (5:33)](images/overview-youtube-image.jpg)](http://www.youtube.com/watch?v=VW1DwDezlqM)
```

Comment:
It strikes me as unconventional to have the Table of Contents not be at the top. I would suggest moving the Overview and Demo to after the Table of Contents and add 'Overview and Demo' as the first item of the Table of Contents.